enclave_manager/avalon_enclave_manager/wpe_common/wpe_requester.py

```diff
@@ -172,6 +168,7 @@ def preprocess_work_order(self, wo_request, encryption_key):
             @returns result - Result from KME that includes the workorder
                               key info. error response, in case of failure.
         """
+        logger.info("inside WPE REQ:: preprocess")
```

Comment:
Probably debug log.

Reply:
Added while debugging functionality. Not really useful, removed it